### PR TITLE
Features/lp 79 add  survey tab

### PR DESCRIPTION
**Pull request by iconico**
Merge of adding the survey tab from Kent's work.
No functional changes, just incorporating code in

### CODE REVIEW

**Review by timelyportfolio — COMMENTED** [review]
@iconico @themightychris I started an initial review here.  I am more than happy to tackle these if easier.  I would appreciate feedback on questions.

File: resources/js/app.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { App, plugin } from '@inertiajs/inertia-vue';
 import { InertiaProgress } from '@inertiajs/progress';
 import Layout from './Shared/Layouts/Layout.vue';
 import vuetify from './plugins/vuetify';
+import store from './store';
 
 import '../scss/app.scss';
 import 'vuetify/dist/vuetify.min.css';
```
[the remark this answers is not in the document]
Comment (themightychris) [reply resources/js/app.js:9]: the main map is using mapboxgl, are we adding leaflet somewhere?

File: resources/js/Pages/Survey/Location.vue [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  computed: {
+    ...mapFields([
+      'blockgroup',
+      'step_current',
+    ]),
+  },
+
+  watch: {
+    step_current(value) {
+      if (value === 2) {
```
[the remark this answers is not in the document]
Comment (iconico) [reply resources/js/Pages/Survey/Location.vue]: fixed in https://github.com/urbanSpatial/LetsPlanOrg/pull/45/commits/cadeb4489e86026f4f570272c1c664e1d340346f

File: resources/js/Pages/Survey/Location.vue [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
... '421010074002', NAME: 'Block Group 2, Census Tract 74, Philadelphia County, Pennsylvania' }, geometry: { type: 'MultiPolygon', coordinates: [[[[-75.212687, 39.944885], [-75.208799, 39.941644], [-75.207838, 39.941286], [-75.21065, 39.939198999999998], [-75.212064, 39.938213], [-75.213865, 39.939974], [-75.21260099999999, 39.94085], [-75.215246, 39.943087], [-75.212687, 39.944885]]]] } }, { type: 'Feature', properties: { GEOID: '421010078001', NAME: 'Block Group 1, Census Tract 78, Philadelphia County, Pennsylvania' }, geometry: { type: 'MultiPolygon', coordinates: [[[[-75.21652, 39.944171], [-75.213961, 39.945974], [-75.211389, 39.947796], [-75.21010299999999, 39.946697], [-75.211401, 39.945802], [-75.212687, 39.944885], [-75.215246, 39.943087], [-75.215615, 39.943402], [-75.21652, 39.944171]]]] } }, { type: 'Feature', properties: { GEOID: '421010088011', NAME: 'Block Group 1, Census Tract 88.01, Philadelphia County, Pennsylvania' }, geometry: { type: 'MultiPolygon', coordinates: [[[[-75.200525, 39.952695], [-75.200098, 39.954581999999998], [-75.196603, 39.954149], [-75.197243, 39.951021], [-75.199204, 39.951257], [-75.200801, 39.951457999999998], [-75.200525, 39.952695]]]] } }] },
```
[the remark this answers is not in the document]
Comment (iconico) [reply resources/js/Pages/Survey/Location.vue:105]: Totally agree

File: resources/js/Pages/Survey.vue [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    const randIndex = window.Math.floor(window.Math.random() * 3);
+    this.survey_id = this.survey_ids[randIndex];
+
+    // async defer
+    surveyScript.setAttribute('src', `https://formfacade.com/include/112462077823067593328/form/${this.survey_id}/classic.js?div=ff-compose`);
+    document.head.appendChild(surveyScript);
+  },
+};
+</script>
+
```
[the remark this answers is not in the document]
Comment (iconico) [reply resources/js/Pages/Survey.vue:111]: you're right - added that in https://github.com/urbanSpatial/LetsPlanOrg/pull/45/commits/5cd6b5a01b98a258def482c8c2dabd4be1ed6439

File: resources/js/app.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (iconico) [reply resources/js/app.js:11]: I've added it in https://github.com/urbanSpatial/LetsPlanOrg/pull/45/commits/d073c304f9e6436ef5c37afe42d227c11680b2a5 we need it for the survey map to render correctly.

File: resources/js/Pages/Survey/Location.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (iconico) [reply resources/js/Pages/Survey/Location.vue]: I split it out as a JSON object into a separate file: https://github.com/urbanSpatial/LetsPlanOrg/pull/45/commits/4ecbce58834fbb387f50d7fc00129661b70542e6

File: resources/js/store/index.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    survey_google_gid: {
+      '1FAIpQLScC5u7-N9Y7FnDbLVYtU4JoD-6EcSH__nwhsuBZqjTW7Su5Dg': '1441370576',
+      '1FAIpQLSf1SVRyc5p6PUReApldNZbTmqBaUnju1LhgW1zloeeID3OVRw': '1286301424',
+      '1FAIpQLSenWJ_YMi_92OmaHyChEz1zJK9Dy_vIEEi5xB6OE6uIdt-vOQ': '1892223559',
+    },
+    survey_results: null,
+    submitted: false,
+
+    // from letsplanorg
+    exploreIsExpanded: true,
```
[the remark this answers is not in the document]
Comment (iconico) [reply resources/js/store/index.js:25]: I'm down for some vuex https://github.com/urbanSpatial/LetsPlanOrg/pull/45/commits/2cac828180d14d0a33fe417737d43b607bdef6f3

**Pull request merged**